Wikipedia/Code/FindAndReplaceKeyboardBar.swift

```diff
@@ -289,17 +304,24 @@ private extension FindAndReplaceKeyboardBar {
         nextButton.isEnabled = total >= 2
     }
     
+    func replaceTypeText() -> String {
```

Comment:
if it makes it easier, this could be a computed `text` property inside `ReplaceType` and then you could do `replaceType.text` or `accessibilityTex`, but either way is 👍

Reply:
good idea, done!